services/web-app/components/inline-error/inline-error.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Column, Grid, Link } from '@carbon/react'
+import {
+  Close,
+  ErrorFilled
+} from '@carbon/react/icons'
+import clsx from 'classnames'
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+
+import styles from './inline-error.module.scss'
+
+export const InlineError = ({ className, content, title, description, href, link }) => {
+  const [showNotification, setShowNotification] = useState(true)
+  const containerClassName = clsx(className, {
+    'cds--inline-notification': true,
+    'cds--inline-notification--low-contrast': true,
+    'cds--inline-notification--error': true
+  })
+
+  const closeNotifcation = () => {
+    setShowNotification(!showNotification)
+  }
+
+  return (
+    showNotification && (
+      <Grid>
+        <Column sm={4} md={6} lg={12} className={clsx(styles.notification, className)}>
+          <div className={containerClassName}>
+            <Grid>
+              <Column sm={3} md={4} lg={9}>
```

Comment:
let's confirm the inside grid specs here with design, I think 8/4 looks better? but the text spans to two lines . Figma also looks like it's 8/4 in the inner grid but it's really hard to read 🤔 
<img width="934" alt="image" src="https://user-images.githubusercontent.com/40550942/180879820-97ae13dd-670a-424c-af07-21c463fe31ee.png">
<img width="932" alt="image" src="https://user-images.githubusercontent.com/40550942/180879923-8c8e5825-cc28-43b7-a405-0c79368c6f90.png">